app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -189,7 +189,8 @@ def create_valkyrie_work
           'work_resource.add_to_parent' => { parent_id: params[:parent_id], user: current_user },
           'work_resource.add_file_sets' => { uploaded_files: uploaded_files, file_set_params: params[hash_key_for_curation_concern][:file_set] },
           'change_set.set_user_as_depositor' => { user: current_user },
-          'work_resource.change_depositor' => { user: ::User.find_by_user_key(form.on_behalf_of) }
+          'work_resource.change_depositor' => { user: ::User.find_by_user_key(form.on_behalf_of) },
+          'work_resource.save_acl' => { permissions_params: form.input_params["permissions"] }
```

Comment:
`form.input_params` is passed instead of the raw params because `reform` transforms the raw params into an array as part of the validation.  We could pass the raw params instead but they will be harder to work with and any other code calling the transaction outside of a controller context would have to format the params into hashes with index keys instead of the more natural arrays.